### ISSUE TITLE
Cannot find module '@babel/code-frame'

### ISSUE THREAD

**Issue by LisBerndt** [Xray-App/playwright-junit-reporter#1]
Hello! In advance, thanks for opening this project!

I followed the instructions, passed the options in my config file, but when executing the tests, it runs immediately into:

``` Error: Cannot find module '@babel/code-frame'
Require stack:
- C:\playwright\node_modules\@xray-app\playwright-junit-reporter\dist\base.js
- C:\playwright\node_modules\@xray-app\playwright-junit-reporter\dist\index.js
- C:\playwright\node_modules\@playwright\test\lib\common\transform.js
- C:\playwright\node_modules\@playwright\test\lib\common\testType.js
- C:\playwright\node_modules\@playwright\test\lib\common\test.js
- C:\playwright\node_modules\@playwright\test\lib\reporters\multiplexer.js
- C:\playwright\node_modules\@playwright\test\lib\runner\reporters.js
- C:\playwright\node_modules\@playwright\test\lib\runner\runner.js
- C:\playwright\node_modules\@playwright\test\lib\cli.js
- C:\playwright\node_modules\playwright-core\lib\cli\cli.js
- C:\playwright\node_modules\playwright-core\cli.js
- C:\playwright\node_modules\@playwright\test\cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1021:15)
    at Function.resolveFilename (C:\playwright\node_modules\@playwright\test\lib\common\transform.js:173:36)
    at Function.Module._load (node:internal/modules/cjs/loader:866:27)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (C:\playwright\node_modules\@xray-app\playwright-junit-reporter\dist\base.js:26:22)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Object.i.<computed>.ot._extensions.<computed> (C:\playwright\node_modules\@playwright\test\lib\utilsBundleImpl.js:16:1010)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (C:\playwright\node_modules\@xray-app\playwright-junit-reporter\dist\index.js:24:16)
    at Module._compile (node:internal/modules/cjs/loader:1191:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1245:10)
    at Object.i.<computed>.ot._extensions.<computed> (C:\playwright\node_modules\@playwright\test\lib\utilsBundleImpl.js:16:1010)
    at Module.load (node:internal/modules/cjs/loader:1069:32)
    at Function.Module._load (node:internal/modules/cjs/loader:904:12)
    at Module.require (node:internal/modules/cjs/loader:1093:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at requireOrImport (C:\playwright\node_modules\@playwright\test\lib\common\transform.js:154:20)
    at requireOrImportDefaultFunction (C:\playwright\node_modules\@playwright\test\lib\runner\loadUtils.js:243:51)
    at loadReporter (C:\playwright\node_modules\@playwright\test\lib\runner\loadUtils.js:252:10)
    at createReporter (C:\playwright\node_modules\@playwright\test\lib\runner\reporters.js:63:71)
    at Runner.runAllTests (C:\playwright\node_modules\@playwright\test\lib\runner\runner.js:64:58)
    at runTests (C:\playwright\node_modules\@playwright\test\lib\cli.js:135:151)
    at ji.<anonymous> (C:\playwright\node_modules\@playwright\test\lib\cli.js:48:7) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\playwright\\node_modules\\@xray-app\\playwright-junit-reporter\\dist\\base.js',
    'C:\\playwright\\node_modules\\@xray-app\\playwright-junit-reporter\\dist\\index.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\common\\transform.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\common\\testType.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\common\\test.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\reporters\\multiplexer.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\runner\\reporters.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\runner\\runner.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\lib\\cli.js',
    'C:\\playwright\\node_modules\\playwright-core\\lib\\cli\\cli.js',
    'C:\\playwright\\node_modules\\playwright-core\\cli.js',
    'C:\\playwright\\node_modules\\@playwright\\test\\cli.js'
  ]
}```

**Comment by bitcoder** [comment]
I'm having the same issue.. suddenly on another project that I have on GH it started giving this exact error. I cannot replicate it locally though, so help is appreciated. I tried enforcing the package.json to include an explicit stable version of playwright (1.33.0) but it hasn't solved it

**Comment by bitcoder** [comment]
it seems related to eslint (?). the generated package/dist/base.js contains that reference.. I need to figure out what is adding it exactly

**Comment by bitcoder** [comment]
Found it..  need a bit to fix it

**Comment by bitcoder** [comment]
fixed it and published version 0.1.2
